blog/2023-10-29-ratify-1.0.0.mdx

```diff
@@ -0,0 +1,93 @@
+---
+slug: ratify-v1.0.0-is-available
+title: Ratify v1.0.0 is available! An extensible verification framework for software supply chain security
+authors: [ratify]
+tags: [verification]
+---
+
+The Ratify maintainers are proud to announce the first major release v1.0.0 for production use!
+
+Ratify is an extensible verification framework for container images and other artifacts that can examine and use custom policies that you create to approve deployments in Kubernetes. Ratify can use and coordinate any number of custom verifiers for image metadata like signatures, SBOMs, vulnerability scan reports, and so on.
```

Comment:
Did you mean "manage" instead of "coordinate"?

Reply:
"manage" is better. Updated